### ISSUE TITLE
GuzzleHttp\Exception\ConnectException: cURL error 7: Failed to connect to www.nextcloud.com port 80: Connection timed out

### ISSUE THREAD

**Issue by boingo-00** [nextcloud/server#10962]
### Steps to reproduce
1. Checking logs
2. cURL from server
3.
_www.nextcloud.com port 80_ is down
```
PING www.nextcloud.com (165.227.162.232) 56(84) bytes of data.
^C
--- www.nextcloud.com ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2050ms
```

### Server configuration

**Operating system**: Debian stable (9)

**Web server:** apache2  2.4.25-3+deb9u5

**Database:** mysql 10.1.26

**PHP version:** 7.0.30

**Nextcloud version:** 13.0.6 (see Nextcloud admin page)

**Updated from an older Nextcloud/ownCloud or fresh install:** upd

**Where did you install Nextcloud from:** `deb http://apt.jurisic.org/debian/ stretch main contrib non-free`

**Comment by patschi** [comment]
Host `www.nextcloud.com` is working fine from here right now - ICMP as well as HTTP/S, so it might have been a temporary issue on the website or a issue on your end. Maybe any firewalls blocking outgoing requests to that specific IP address? However, this is probably a bit out-of-scope of the PHP code of Nextcloud server anyway. That's a better place to ask for help in this case: https://help.nextcloud.com

**Comment by boingo-00** [comment]
I checked site right now, ping not working. Browser can open site, but curl can't. I temporary disabled firewall, but site not working. Maybe site blocked by RKN?
----------------------------------------
Checked logs, error don't repeated after opening issue. Magic?

**Comment by Drugoy** [comment]
Поиск по IP на 2018-09-10 22:19:45:
❌ 165.227.162.232 (www.nextcloud.com) заблокирован, идентификатор записи #888246
